backend/src/main/java/ch/puzzle/bbt/puzzlescrumpoker/PokerService.java

```diff
@@ -56,6 +56,9 @@ public String getSelectedCard(String gamekey, long playerid) throws Exception {
 
     public void setPlayerMode(String gamekey, long playerid, boolean playerMode) throws Exception {
         getTableById(gamekey).getPlayerById(playerid).setPlaying(playerMode);
+        if (playerMode) {
```

Comment:
Kann man den Wert nicht schon löschen, wenn der Player in den Spectator-Mode geht?
-> if ( ! playerMode) {